docker/Dockerfile

```diff
@@ -23,13 +23,15 @@
 #
 ##############################################################
 
-FROM python:3.7
+ARG base=python:3.9-slim-bullseye
+FROM $base
 
-RUN apt-get update && apt-get install -y -q \
-    mpich \
-    libmpich-dev \
-    libgl1-mesa-glx
+# Install for basic base not containing it.
```

Comment:
this is a bit voodoo, can we rather further split Dockerfile?

Reply:
I'd rather not it would just copy paste all this one except one line